homeassistant/components/emoncms/manifest.json

```diff
@@ -4,5 +4,5 @@
   "documentation": "https://www.home-assistant.io/integrations/emoncms",
   "requirements": [],
   "dependencies": [],
-  "codeowners": []
+  "codeowners": ["@borpin"]
```

Comment:
Please add yourself to the [codeowners file](https://github.com/home-assistant/home-assistant/blob/dev/CODEOWNERS).